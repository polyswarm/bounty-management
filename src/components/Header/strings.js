const strings = {
  newBounty: '+ Bounty',
};

export default strings;
