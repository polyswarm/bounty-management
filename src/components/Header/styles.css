.Header {
  background-color: #8500ff;
  box-shadow: 0 1px 1px black;
  color: white;
}
