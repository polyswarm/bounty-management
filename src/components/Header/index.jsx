import React, { Component } from 'react';
import './styles.css';

class Header extends Component {

  render() {
    const { props: { title } } = this;
    return (
      <header className='Header'>
        <h1>{title}</h1>
      </header>
    );
  }

}

export default Header;
