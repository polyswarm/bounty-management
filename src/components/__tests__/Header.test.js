import React from 'react';
import {render} from 'enzyme';
import {renderToJson} from 'enzyme-to-json';
import Header from '../Header';

it('renders without crashing', () => {
  const wrapper = render(<Header />);
  expect(renderToJson(wrapper)).toMatchSnapshot();
});
