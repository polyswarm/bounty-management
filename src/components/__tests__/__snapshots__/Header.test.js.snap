// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<header
  class="Header"
>
  <h1>
    
  </h1>
</header>
`;
